docs/source/workflows/observe/index.md

```diff
@@ -17,12 +17,14 @@ limitations under the License.
 
 # Observe Workflows
 
-The NeMo Agent toolkit Observability Module provides support for configuring logging, tracing, and metrics for NeMo Agent toolkit workflows. Users can configure telemetry options from a predefined list based on their preferences. The logging and tracing exporters:
+The NeMo Agent toolkit uses a flexible, plugin-based observability system that provides comprehensive support for configuring logging, tracing, and metrics for workflows. Users can configure multiple telemetry exporters simultaneously from the available options or create custom integrations. The observability system:
 
-- Listen for usage statistics pushed by `IntermediateStepManager`.
-- Translate the usage statistics to OpenTelemetry format and push to the configured provider/method. (e.g., phoenix, OTelCollector, console, file)
+- Uses an event-driven architecture with `IntermediateStepManager` publishing workflow events to a reactive stream
+- Supports multiple concurrent telemetry exporters processing events asynchronously
+- Provides built-in exporters for popular observability platforms (Phoenix, Langfuse, Weave, etc.)
+- Enables custom telemetry exporter development for any observability service
 
-These features enable NeMo Agent toolkit developers to test their workflows locally and integrate observability seamlessly.
+These features enable NeMo Agent toolkit developers to test their workflows locally and integrate observability seamlessly with their preferred monitoring stack.
```

Comment:
Consider:
```suggestion
These features enable developers to test their workflows locally and integrate observability seamlessly with their preferred monitoring stack.
```

...because our name is too long to repeat sometimes, feel free to ignore this suggestion as well